### PR TITLE
Add mitol-django-common

### LINKED ISSUE #9: Utilize mitol-django-common
We should pull in https://github.com/mitodl/ol-django/tree/main/mitol-django-common (https://pypi.org/project/mitol-django-common/) so we can start using the common data models and utilities from there.

### PR DESCRIPTION
**Pull request by rhysyngsun**
#### Pre-Flight checklist

- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested
- [x] Settings
  - [x] New settings are documented and present in `app.json`
  - [x] New settings have reasonable development defaults, if applicable

#### What are the relevant tickets?
Closes #9 

#### What's this PR do?
- Add `isort` check to github actions + ran it on the code
- Incorporates `mitol-django-common`
- Removes a number of functionality that is now in that library:
  - `main.envs`
  - `main.templatetags.render_bundle`
  - `main.utils`
- Ran `./manage.py generate_app_json`, which means everything in `app.json` gets a description now.

#### How should this be manually tested?
The app should start and you should be able to login to django-admin

### CODE REVIEW

File: main/apps.py [hunk 1]
```diff
@@ -1,26 +1,18 @@
...
-            raise ImproperlyConfigured(
-                "The following settings are missing: {}".format(
-                    ", ".join(missing_settings)
-                )
-            )
+    required_settings: List[str] = [
+        "MAILGUN_KEY",
+        "MAILGUN_SENDER_DOMAIN",
+        "SECRET_KEY",
+    ]
```
Comment (mbertrand) [inline main/apps.py:18]: I removed MAILGUN_* from my .env and no warning/errors showed up, should they?

**Review by mbertrand — COMMENTED** [review]
Looks good, 1 question about required settings

File: main/apps.py (hunk 1 above)
Comment (rhysyngsun) [reply main/apps.py]: They should but the don't appear to be - this also seems to duplicate the behavior of `envs.validate()`, so i just removed this and put in a call that that instead. I'll need to figure out later why I seemingly built in 2 similar mechanisms into that library...

**Comment by rhysyngsun** [comment]
@mbertrand all set finally

**Comment by rhysyngsun** [comment]
Rebased, but i had to squash to do so

File: ocw_import/tasks.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 import logging
 
 import celery
+from mitol.common.utils.collections import chunks
 
 from main.celery import app
 from main.utils import chunks
```
Comment (mbertrand) [inline ocw_import/tasks.py:8]: Need to remove this import

**Review by mbertrand — CHANGES_REQUESTED** [review]
Works great, just one stray import

**Review by mbertrand — APPROVED** [review]
👍

**Pull request merged**